cmd/machine-config-daemon/firstboot_complete_machineconfig.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	daemon "github.com/openshift/machine-config-operator/pkg/daemon"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var firstbootCompleteMachineconfig = &cobra.Command{
+	Use:                   "firstboot-complete-machineconfig",
+	DisableFlagsInUseLine: true,
+	Short:                 "Complete the host's initial boot into a MachineConfig",
+	Args:                  cobra.MaximumNArgs(0),
+	Run:                   executeFirstbootCompleteMachineConfig,
+}
+
+// init executes upon import
+func init() {
+	rootCmd.AddCommand(firstbootCompleteMachineconfig)
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+}
+
+func runFirstBootCompleteMachineConfig(_ *cobra.Command, _ []string) error {
+	flag.Set("logtostderr", "true")
+	flag.Parse()
+
+	exitCh := make(chan error)
+	defer close(exitCh)
+
+	dn, err := daemon.New(daemon.NewNodeUpdaterClient(), exitCh)
+	if err != nil {
+		return err
+	}
+
+	err = dn.RunFirstbootCompleteMachineconfig()
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
nit: unnecessary error check (can do simply `return dn.RunFirstbootCompleteMachineconfig()`)

Reply:
Nice catch, fixed it!